---
title: "Allium cepa analysis"
description: |
  Page to report on the reanalysis of a RAD-seq dataset of onions
output:
  distill::distill_article:
    self_contained: false
    toc: true
    toc_depth: 3
bibliography: references/allium_cepa_reanalysis_refs.bib
csl: references/ref_style/acm-sig-proceedings.csl
cite-method: citeproc
---

# _Allium cepa_ RAD-seq dataset reanalysis

Reanalysis of the dataset from Lee _et al_ (2018) [@lee2018snp] using stacks and a reference genome that was not included in [@lee2018snp] as said genome was published well after Lee published its RAD-seq analysis. 

## Context

This reanalysis was done with the purpose of testing some software before using it in on dataset that i expected to get. The main focus is to try to find population structure in those datasets. I also wanted to study some concepts of population genetics here. This is not necessarily an state of the art reanalysis of the dataset. 

## Data description and collection

### Raw reads

The raw reads consist of 192 Illumina PE libraries. Each library corresponds to an individual from four korean companies [@lee2018snp]. The reads were retrieved with [SRA-Explorer](http://sra-explorer.info/) [@sraexplorer], using the SRA(Sequence read archive) accesion code: SRP150117.

### Reference genome

The reference genome was downloaded from the NCBI with assembly accesion code: GCA_905187595.1. The associated paper is [@finkers2021insights]. 

## Softwares used 

- Stacks V2.64
- fastp  V0.23.4
- Samtools


## Pipeline run

### Reference guided genotyping


### deNovo genotyping



# References

::: {#refs}
:::
